fictadvisor-api/src/v2/api/controllers/UserController.ts

```diff
@@ -812,6 +812,34 @@ export class UserController {
   }
 
   @Access(PERMISSION.USERS_$USERID_TELEGRAM_LINK)
+  @ApiOkResponse({
+    type: OrdinaryStudentResponse,
+    description: 'Added user\'s telegram link',
+  })
+  @ApiBadRequestResponse({
+    description: `\n
+    InvalidEntityIdException:
+      User with such id is not found`,
+  })
+  @ApiForbiddenResponse({
+    description: `\n
+    NoPermissionException:
+      You do not have permission to perform this action`,
+  })
+  @ApiParam({
+    name: 'userId',
+    required: true,
+    description: 'Id of a user',
+  })
+  @ApiParam({
+    name: 'telegram',
+    required: true,
+    description: 'Telegram link of a user',
+  })
```

Comment:
This endpoint takes only 1 parameter userId, telegram is not parameter it is a body